multivac/gather_data.py

```diff
@@ -318,7 +326,8 @@ def sorter(filename):
             except FileNotFoundError:
                 print(f'No logs for job {job_id}, {job["html_url"]}')
             else:
-                time_queued = log_file_as_list[0][0:19] + 'Z'
+                time_queued = get_log_datetime(log_file_as_list[0]) \
```

Comment:
Add a comment on why the value is passed at 0, and not just as is (list)

Reply:
Done